packages/@react-stately/color/src/useColorWheelState.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Color} from './Color';
+import {ColorWheelProps} from '@react-types/color';
+import {useControlledState} from '@react-stately/utils';
+import {useState} from 'react';
+
+export interface ColorWheelState {
+  readonly value: Color,
+  setValue(value: string | Color): void,
+
+  readonly hue: number,
+  setHue(value: number): void,
+
+  dragging: boolean,
```

Comment:
Usually try to prefix booleans with `is`/`should` etc. I think `isDragging` in this case.